eth-contracts/contracts/DelegateManager.sol

```diff
@@ -1022,7 +1022,6 @@ contract DelegateManager is InitializableV2 {
 
         // Amount stored in staking contract for owner
         totalBalanceInStaking = Staking(stakingAddress).totalStakedFor(_serviceProvider);
-        require(totalBalanceInStaking > 0, "DelegateManager: Stake required for claim");
```

Comment:
curious what this does if someone who has never staked before attempts a claim

Reply:
yes! i modified a test case here to confirm this behavior https://github.com/AudiusProject/audius-protocol/pull/883/files#diff-befa24b9618f4a451170554d1b92bcffR557

Reply:
👍